Makefile

```diff
@@ -113,9 +113,10 @@ lint: golangci-lint
 	GOLANGCI_LINT_CACHE=/tmp/golangci-cache $(GOLANGCI_LINT) run
 
 UNIT_TEST_DIRS=$(shell go list ./... | grep -v /test/)
+ENVTEST_OPTS ?= $(if $(OPENSHIFT_CI),--bin-dir=/tmp)
```

Comment:
I realize that OPENSHIFT_CI could be set to "false" as we're not checking the value of that variable. That said, the existence of this variable should be sufficient for now given local environments are likely setting it, and CI is guaranteed to have it specified.